readme.md

```diff
@@ -1,10 +1,12 @@
 
 # OHM DELIVERY
-ohm-delivery is a minimalistic project that you have to improve and build. 
+This is a fictinal company minimalistic project that you have to fix and improve.
 
-The final customer receive the following trackingId `1e62adfe` by email and a link to the tracking page. 
+The company delivers electrical resitances and provides a tracking page to their drivers.
```

Comment:
**Question:** is this really only provided to their drivers? In my mind it was the original customer, despite the scope is bit mixed up I agree. Maybe we could say to their drivers and customers?